halle

```diff
@@ -4,13 +4,20 @@ asn: 65046
 networks:
   ipv4:
     - 10.62.0.0/16
+  ipv6:
+    - fd0f:8db3:d50a::/64
```

Comment:
You're adding an IPv6 network and nameservers but no IPv6 BGP Peers, this will cause breakage.